tests/docs/test_autodoc_coverage.py

```diff
@@ -27,7 +28,11 @@ def all_rst_content():
 
 def public_class_names():
     all_attrs = dir(pushsource)
-    return [elem for elem in all_attrs if not elem.startswith("_")]
```

Comment:
You probably won't need this change after the import of 'util' module is removed.